benchmark_test.go

```diff
@@ -915,3 +915,89 @@ func BenchmarkBitmapReuseWithClear(b *testing.B) {
 		}
 	}
 }
+
+func BenchmarkAndAny(b *testing.B) {
+
```

Comment:
Unnecessary empty line (minor)

Reply:
Thanks for the review. I will handle it.